invisible_cities/reco/icaro_components_test.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+import numpy          as np
+import numpy.testing  as npt
+import pandas         as pd
+import pandas.testing as pdt
+
+from hypothesis             import given
+from hypothesis.strategies  import floats
+
+from .. reco                import icaro_components as icarcomp
+from ..types.symbols        import KrFitFunction
+from .. evm.ic_containers   import FitFunction
+
+
+@given(floats(min_value= 0, max_value= 10),
+       floats(min_value=10, max_value= 20),
+       floats(min_value= 1, max_value=100),
+       floats(min_value= 0, max_value= 10))
+def test_lin_function_output_values(x_min, x_max, a, b):
+    x = np.array([x_min, x_max])
+    y = a + b * x
+    a_test, b_test = icarcomp.lin_seed(x, y)
+    npt.assert_allclose(a_test, a)
+    npt.assert_allclose(b_test, b)
```

Comment:
to test single values use `assert np.isclose`. Use `assert_allclose` for arrays. This applies to all tests.